pyproject.toml

```diff
@@ -49,6 +49,10 @@ vis = [
     "jupyter-bokeh",
 ]
 
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "--durations=10" 
```

Comment:
Please add this option to the invocation of pytest instead, no point in having it as a project property.

Reply:
I prefer it configured once since then every time I invoke `pytest` - either locally or in CI it'll have the same params.